### PR TITLE
Sharding of zip files

### PR DESCRIPTION
**Pull request by ThinkFlipp**

### CODE REVIEW

**Comment by ThinkFlipp** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;161R4G9e&#x2F;206-sharding-of-zip-files"> Sharding of zip-files</a>

File: bin/go-import [hunk 1]
```diff
@@ -99,14 +103,21 @@ class GoImportCommandLine < Thor
...
 
-            go_data_zip = options.output.nil? == true ? "go.zip" : options.output
-            go_files = options.output_documents.nil? == true ? nil : ::File.basename(options.output_documents,File.extname(options.output_documents))
-            model.save_to_zip(go_data_zip, go_files)
-            puts "Source has been been converted into '#{go_data_zip}'."
-            puts "  - and files into '#{go_files}.zip'." if !go_files.nil?
-            if !warnings_msg.empty? 
-                puts "WARNINGS: "
-                puts warnings_msg
+            puts "Starting charding of model..."
```
Comment (petterl) [inline bin/go-import]: charding -> sharding

File: lib/go_import/shard_helper.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            attr_accessor :shards, :current_shard_count, :current_shard
+            
+            def initialize(shard_size = nil)
+                @shard_size = shard_size || 25000
+                setup()
+            end
+
+            def shard_model(model)
+                @current_shard.configuration = model.configuration
+                #@current_shard.coworkers = model.coworkers
```
Comment (palsson) [inline lib/go_import/shard_helper.rb]: Delete commented out code.

**Comment by palsson** [comment]
Looks good. :)

**Pull request merged**